src/app/BUILD.gn

```diff
@@ -208,6 +208,7 @@ static_library("app") {
     "${chip_root}/src/protocols/secure_channel",
     "${chip_root}/src/system",
     "${nlio_root}:nlio",
+    "icd/:icd-server-srcs",
```

Comment:
all the other places use a path. this should be:

"${chip_root}/src/app/icd:icd-server-srcs"

Reply:
done